tests/net/socket/getaddrinfo/src/main.c

```diff
@@ -269,15 +269,38 @@ void test_getaddrinfo_num_ipv4(void)
 	struct sockaddr_in *saddr;
```

Comment:
Tests look ok, but please put them in the separate commit. So one commit for the actual fix, and the other commit for tests.

Reply:
Sure.
Separated to two commits now.